### PR TITLE
Add blog post announcing Comet 0.1.0

### PR DESCRIPTION
**Pull request by andygrove**

### CODE REVIEW

File: _posts/2024-07-20-datafusion-comet-0.1.0.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache DataFusion PMC is pleased to announce the first official source release of the Comet subproject.
```
Comment (alamb) [inline _posts/2024-07-20-datafusion-comet-0.1.0.md:28]: I recommend adding a link to the homepage https://datafusion.apache.org/comet/

Something like 


```suggestion
The Apache DataFusion PMC is pleased to announce the first official source release of the [Comet](https://datafusion.apache.org/comet/) subproject.
```

Perhaps

File: _posts/2024-07-20-datafusion-comet-0.1.0.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+improved performance and efficiency without requiring any code changes.
+
+Comet runs on commodity hardware and aims to provide 100% compatibility with Apache Spark. Any operators or 
+expressions that are not fully compatible will fall back to Spark unless explicitly enabled by the user. Refer 
+to the [compatibility guide] for more information.
+
+[compatibility guide]: https://datafusion.apache.org/comet/user-guide/compatibility.html
+
+This release covers five months of development work since the project was [donated] to the Apache DataFusion 
+project and is the result of merging 343 PRs from 41 contributors. See the [change log] for more information.
```
Comment (alamb) [inline _posts/2024-07-20-datafusion-comet-0.1.0.md:40]: wow -- 41 contributors is quite impressive 🏆

**Review by alamb — APPROVED** [review]
Looks great to me -- thanks @andygrove

**Comment by andygrove** [comment]
Thanks for the review @alamb. I plan on merging this later today once the release is available.

**Comment by viirya** [comment]
Still in draft?

**Review by viirya — APPROVED** [review]

File: _posts/2024-07-20-datafusion-comet-0.1.0.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+expressions that are not fully compatible will fall back to Spark unless explicitly enabled by the user. Refer
+to the [compatibility guide] for more information.
+
+[compatibility guide]: https://datafusion.apache.org/comet/user-guide/compatibility.html
+
+This release covers five months of development work since the project was [donated] to the Apache DataFusion
+project and is the result of merging 343 PRs from 41 contributors. See the [change log] for more information.
+
+[donated]: https://datafusion.apache.org/blog/2024/03/06/comet-donation/
+[change log]: https://github.com/apache/datafusion-comet/blob/changelog-0.1.0/dev/changelog/0.1.0.md
```
Comment (viirya) [inline _posts/2024-07-20-datafusion-comet-0.1.0.md:43]: This link is 404 currently.

File: _posts/2024-07-20-datafusion-comet-0.1.0.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+This first release supports 15 data types, 13 operators, and 106 expressions. Comet is compatible with Apache
```
Comment (viirya) [inline _posts/2024-07-20-datafusion-comet-0.1.0.md:45]: Can we provide a document link to what are actually supported here?

**Pull request merged**